ios/Podfile.lock

```diff
@@ -190,11 +190,11 @@ PODS:
   - nanopb/encode (2.30908.0)
```

Comment:
if the device is locked/eth app is not open, the "finish importing" button is unresponsive on first click. also, if you exit the flow after pairing has occurred but the device is locked/eth app is not open, it glitches out and says "somethign went wrong" next time you enter the flow